src/pages/index.tsx

```diff
@@ -0,0 +1,11 @@
+import type { NextPage } from "next"
+
+const Dashboard: NextPage = () => {
+  return (
+    <main>
```

Comment:
normally I would add <main> in either _app.tsx or in the layout itself since every page would need a <main>